Source/Model/SPTooltip.h

```diff
@@ -30,9 +30,10 @@
 
 #import <WebKit/WebKit.h>
```

Comment:
I know this has already been merged, but can we drop the webkit import here when we switch to WKWebView?

Reply:
It's still part of [WebKit](https://developer.apple.com/documentation/webkit).